products/bash/release-tracing.sh

```diff
@@ -36,8 +35,9 @@ namespace="cp4i"
 release_name="tracing-demo"
 block_storage="ibmc-block-gold"
 file_storage="ibmc-file-gold-gid"
+tracingEnabled="false"
```

Comment:
I don't think it's worth adding an option for this. The earlier lines to exit without releasing tracing are here temporarily to disable tracing for 1-click until we've had a chance to re-enable now we've moved to 2020.3.1. I think generally we would just not call this script if we don't want tracing. Could you revert the changes to this file? We'll re-enable tracing as part of the work on this branch:
https://github.com/IBM/cp4i-deployment-samples/tree/reenable-tracing

Reply:
changes applied